CMakeLists.txt

```diff
@@ -387,29 +387,26 @@ endforeach()
 
 # Find Lua interpreter
 include(LuaHelpers)
-set(LUA_DEPENDENCIES lpeg mpack bit)
 if(NOT LUA_PRG)
   foreach(CURRENT_LUA_PRG luajit lua5.1 lua5.2 lua)
     # If LUA_PRG is set find_program() will not search
     unset(LUA_PRG CACHE)
-    unset(LUA_PRG_WORKS)
     find_program(LUA_PRG ${CURRENT_LUA_PRG})
 
     if(LUA_PRG)
-      check_lua_deps(${LUA_PRG} "${LUA_DEPENDENCIES}" LUA_PRG_WORKS)
```

Comment:
Oh, I see what you're saying now. I'll definitely fix this tonight, thanks for the review and explanation!